Sources/Player/Player.docc/Articles/optimization/optimization-article.md

```diff
@@ -0,0 +1,62 @@
+# Optimization
+
+@Metadata {
+    @PageColor(purple)
+    @PageImage(purpose: card, source: optimization-card, alt: "An image depicting a speedometer.")
+}
+
+Avoid wasting system resources unnecessarily.
+
+## Overview
+
+Applications must use system resources—such as CPU, memory, and network bandwidth—efficiently to avoid excessive memory consumption (which could lead to app termination) and to conserve battery life.
+
+Media playback is frequently highlighted in product specifications as a key indicator of battery performance. This is no coincidence, as activities like video streaming involve significant resource demands: the network interface must wake periodically to download content, the processor must decode it, and the screen must remain active to display it to the user.
+
+By ensuring your application manages resources responsibly, especially when handling video or audio playback, you enhance the user experience. Not only will users be able to enjoy your content for longer, but they’ll also extend their device’s battery life, enabling more use on a single charge.
+
+This article discusses a few strategies to reduce resource consumption associated with PillarboxPlayer in your application.
```

Comment:
```suggestion
This article discusses a few strategies to reduce resource consumption associated with ``PillarboxPlayer`` in your application.
```

Reply:
``PillarboxPlayer`` is not a symbol. The package was meant here but of course we could point at ``Player`` instead.

Reply:
Yes, I understand, but since other occurrences of ``PillarboxPlayer`` are used in the article, I suggested this for consistency.

Reply:
You're right, I made mistakes with other references. I'll check the documentation and make some fixes, thanks.